en/services/gimbal_v2.md

```diff
@@ -0,0 +1,282 @@
+
+
+#### Onboard computer with camera and gimbal conntected to autopilot
+
+In this set-up the gimbal manager can be on the onboard computer.
+For this case all commands for gimbal manager need to be sent to the gimbal manager (in this case the companion computer), and the messages to the gimbal device need to be sent/routed to the autopilot.
+
+<!-- Mermaid graph: 
+graph LR
+	ap["Autopilot"]
+	cc["Companion (Gimbal Manager)"]
+	g["Gimbal Device"]
+	gcs["Ground Station"]
+	ap -- >|GIMBAL_DEVICE_SET_ATTITUDE|g
+	g -- >|GIMBAL_DEVICE_ATTITUDE_STATUS|ap
+	ap -- >|"DO_GIMBAL_MANAGER_ATTITUDE<br/>DO_SET_ROI_LOCATION<br/>(commands in mission)"| cc
+	ap -- >|GIMBAL_DEVICE_ATTITUDE_STATUS|cc
+	gcs -- >|GIMBAL_MANAGER_SET_ATTITUDE|cc
+	cc -- >|GIMBAL_MANAGER_INFORMATION<br/>GIMBAL_MANAGER_STATUS<br/>GIMBAL_DEVICE_ATTITUDE_STATUS|gcs
+	cc -- >|GIMBAL_DEVICE_SET_ATTITUDE|ap
+-->
+
+[![](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggTFJcblx0YXBbXCJBdXRvcGlsb3RcIl1cblx0Y2NbXCJDb21wYW5pb24gKEdpbWJhbCBNYW5hZ2VyKVwiXVxuXHRnW1wiR2ltYmFsIERldmljZVwiXVxuXHRnY3NbXCJHcm91bmQgU3RhdGlvblwiXVxuXHRhcCAtLT58R0lNQkFMX0RFVklDRV9TRVRfQVRUSVRVREV8Z1xuXHRnIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xhcFxuXHRhcCAtLT58XCJET19HSU1CQUxfTUFOQUdFUl9BVFRJVFVERTxici8-RE9fU0VUX1JPSV9MT0NBVElPTjxici8-KGNvbW1hbmRzIGluIG1pc3Npb24pXCJ8IGNjXG5cdGFwIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xjY1xuXHRnY3MgLS0-fEdJTUJBTF9NQU5BR0VSX1NFVF9BVFRJVFVERXxjY1xuXHRjYyAtLT58R0lNQkFMX01BTkFHRVJfSU5GT1JNQVRJT048YnIvPkdJTUJBTF9NQU5BR0VSX1NUQVRVUzxici8-R0lNQkFMX0RFVklDRV9BVFRJVFVERV9TVEFUVVN8Z2NzXG5cdGNjIC0tPnxHSU1CQUxfREVWSUNFX1NFVF9BVFRJVFVERXxhcCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggTFJcblx0YXBbXCJBdXRvcGlsb3RcIl1cblx0Y2NbXCJDb21wYW5pb24gKEdpbWJhbCBNYW5hZ2VyKVwiXVxuXHRnW1wiR2ltYmFsIERldmljZVwiXVxuXHRnY3NbXCJHcm91bmQgU3RhdGlvblwiXVxuXHRhcCAtLT58R0lNQkFMX0RFVklDRV9TRVRfQVRUSVRVREV8Z1xuXHRnIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xhcFxuXHRhcCAtLT58XCJET19HSU1CQUxfTUFOQUdFUl9BVFRJVFVERTxici8-RE9fU0VUX1JPSV9MT0NBVElPTjxici8-KGNvbW1hbmRzIGluIG1pc3Npb24pXCJ8IGNjXG5cdGFwIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xjY1xuXHRnY3MgLS0-fEdJTUJBTF9NQU5BR0VSX1NFVF9BVFRJVFVERXxjY1xuXHRjYyAtLT58R0lNQkFMX01BTkFHRVJfSU5GT1JNQVRJT048YnIvPkdJTUJBTF9NQU5BR0VSX1NUQVRVUzxici8-R0lNQkFMX0RFVklDRV9BVFRJVFVERV9TVEFUVVN8Z2NzXG5cdGNjIC0tPnxHSU1CQUxfREVWSUNFX1NFVF9BVFRJVFVERXxhcCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)
+
+
+### Message flow
```

Comment:
This section might reasonably go up as part of the first concept when you describe the different components.

Reply:
Done.